src/main/java/com/google/idea/perf/TracerView.kt

```diff
@@ -97,6 +103,15 @@ class TracerView(parentDisposable: Disposable) : JBPanel<TracerView>() {
         // Call list.
         add(JBScrollPane(listView))
 
+        // Render time label.
+        refreshTimeLabel = JBLabel().apply {
+            font = Font(Font.MONOSPACED, font.style, font.size)
```

Comment:
JetBrains has a custom Font factory method that we should try using here:
```kotlin
JBUI.Fonts.create(Font.MONOSPACED, font.size)
```
I'm not sure exactly what the difference is, but it looks like it has better handling for font scaling (maybe for things like IntelliJ's presentation mode, not sure).

Reply:
Thanks, I've been looking for this!